you-should-use.plugin.zsh

```diff
@@ -273,7 +360,16 @@ function enable_you_should_use() {
     add-zsh-hook preexec _check_global_aliases
     add-zsh-hook preexec _check_git_aliases
     add-zsh-hook precmd _flush_ysu_buffer
+    if zstyle -T ':you-should-use:*' you_should_use_abbreviation_enabled; then
+        add-zsh-hook preexec _check_abbrs
+    fi
 }
 
+zstyle ':you-should-use:*' you_should_use_alias_enabled true
+zstyle ':you-should-use:*' you_should_use_abbreviation_enabled false
```

Comment:
for some reason tip for abbreviations refuse to show
`zstyle ':you-should-use:*' you_should_use_abbreviation_enabled false`
this is set in the plugin file

even if `zstyle ':you-should-use:*' you_should_use_abbreviation_enabled true` is set in `~/.zshrc` or ran interactively 
and `zstyle ':you-should-use:*' you_should_use_abbreviation_enabled` by default is false anyway so it this necessary?
```suggestion
# zstyle ':you-should-use:*' you_should_use_abbreviation_enabled false
```